File: Strong Number (Special Numbers Series #2).ts
export function factorial(num:number):number {
 if (num < 2) return 1;
 return num * factorial(num - 1);
}

export function strongNumber(n:number):string {
  return String(n).split("").map(i => factorial(Number(i))).reduce((a,b) => a+b,0)===n ? "STRONG!!!!" : "Not Strong !!";
}

File: Strong Number (Special Numbers Series #2).dart
int factorial(int n){
  if(n<2)return 1;
  return n*factorial(n-1);
}

String strong(int n) {
  var sum =n.toString().split("").map((i) => factorial(int.parse(i))).fold(0,(int a,int b) => a+b);
  return sum == n ? "STRONG!!!!" : "Not Strong !!";
}
